src/Mutation.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Softonic\GraphQL;
+
+use Softonic\GraphQL\Config\MutationConfigObject;
+use Softonic\GraphQL\Mutation\Item;
+use Softonic\GraphQL\Query\Item as QueryItem;
+
+class Mutation
+{
+    private $mutation;
+
+    private $config;
+
+    /**
+     * @param array<MutationConfigObject> $mutationConfig
+     */
+    public function __construct(array $mutationConfig, QueryItem $source)
```

Comment:
`$mutationsConfig` instead?

Reply:
No, mutation will be an object with all the variables of the mutation.